doc/source/reference/random/extending.rst

```diff
@@ -5,24 +5,23 @@
 Extending
 =========
 The BitGenerators have been designed to be extendable using standard tools for
-high-performance Python -- numba and Cython.  The `~Generator` object can also
+high-performance Python -- Numba and Cython.  The `~Generator` object can also
 be used with user-provided BitGenerators as long as these export a small set of
 required functions.
 
 Numba
 -----
-Numba can be used with either CTypes or CFFI.  The current iteration of the
-BitGenerators all export a small set of functions through both interfaces.
+Numba supports usage of `~Generator` object, alternatively users can use CFFI.
+The current iteration of the BitGenerators all export a small set of functions
+through both interfaces.
 
-This example shows how numba can be used to produce gaussian samples using
-a pure Python implementation which is then compiled.  The random numbers are
-provided by ``ctypes.next_double``.
+This example shows how Numba can be used to produce random samples using
+a pure Python implementation which is then compiled.
```

Comment:
I agree this should be split into two examples: the previous one which shows how to create a new random sampling function, and the new one to show how to Numba-JIT a generator.

Reply:
Addressed in [1f6172a](https://github.com/numpy/numpy/pull/22109/commits/1f6172a51a19fc1662f0ae06d1cd96f0657f02a9)